docs/lang/articles/misc/gui.md

```diff
@@ -344,20 +176,8 @@ For example, loop until ESC is pressed:
         gui.set_image(img)
         gui.show()
 
-[`gui.get_events(a, ...)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=get_event#taichi.misc.gui.GUI.get_events)
-is basically the same as `gui.get_event`, except that it returns a generator of events instead of storing into `gui.event`:
-
-    for e in gui.get_events():
-        if e.key == ti.GUI.ESCAPE:
-            exit()
-        elif e.key == ti.GUI.SPACE:
-            do_something()
-        elif e.key in ['a', ti.GUI.LEFT]:
-            ...
-
-[`gui.is_pressed(key, ...)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=is_pressed#taichi.misc.gui.GUI.is_pressed)
-can detect the keys you pressed. It must be used together with `gui.get_event`, or it won't be updated! For
-example:
+`gui.is_pressed(key, ...)` detects the keys you pressed. You must use it
+together with `gui.get_event`. Otherwise, it is not updated. For example:
```

Comment:
Will `it won't be updated` be  more clear?

Reply:
This is simply a style issue. Avoid future tense and contraction in technical writing.